recirq/time_crystals/dtc_utilities.py

```diff
@@ -0,0 +1,236 @@
+# Copyright 2021 Google
```

Comment:
try to avoid "utilities" in module names. It's not very descriptive. Try to choose a more descriptive name